src/components/nav/nav.module.scss

```diff
@@ -22,7 +22,7 @@
 
 .logo {
   display: block;
-  width: 150px;
+  width: 250px;
```

Comment:
> These changes seem to have fixed the logo image margins!

It's this diff specifically that adjusted the margins by changing the width on the logo—I think there's still a _slight_ difference between the left and right margins on the logo, but @jayuhmee seemed ok with the current state, and I trust her take more than my own 😅

Reply:
Ahhh honestly didn't even realize that until you pointed it out!